### ISSUE TITLE
Uneven edge of the CLI table

### ISSUE THREAD

**Issue by RobinTail** [arethetypeswrong/arethetypeswrong.github.io#90]
Not a big issue, but I was wondering why it looks like this having so advanced libraries used for the CLI formatting and colorizing.
My best guess is that emojies are UTF-16, while the tool making paddings only accepts UTF-8.
Perhaps there should be some special method that can figure out UTF-16 string length in order to make correct paddings.

![image](https://github.com/arethetypeswrong/arethetypeswrong.github.io/assets/13189514/ec0fbf53-fbdf-4301-a79d-6b9d808752a7)

It's also interesting that the space after green circle is only visible when I copy/paste the text from the console.
I'm using [iTerm2](https://iterm2.com/).

```
┌───────────────────┬───────────────────┐
│                   │ "express-zod-api" │
├───────────────────┼───────────────────┤
│ node10            │ 🟢                │
├───────────────────┼───────────────────┤
│ node16 (from CJS) │ 🟢 (CJS)          │
├───────────────────┼───────────────────┤
│ node16 (from ESM) │ 🟢 (ESM)          │
├───────────────────┼───────────────────┤
│ bundler           │ 🟢                │
└───────────────────┴───────────────────┘
```

**Comment by andrewbranch** [comment]
I have also noticed this and it bugs me, but not enough to try to track down a fix in cli-table3. https://github.com/cli-table/cli-table3. Maybe a small repro there would gain some traction. I’d love to pick up a fix but don’t have time to do it myself.

**Comment by RobinTail** [comment]
Fixed in v0.13.0, @andrewbranch 

![image](https://github.com/arethetypeswrong/arethetypeswrong.github.io/assets/13189514/190e1ea6-86ba-4d5c-a648-f5f89faf4cd6)

**Comment by andrewbranch** [comment]
Huhmmm, if that’s true, I guess it wasn’t cli-table3 because they haven’t had any new releases: https://github.com/cli-table/cli-table3/releases. In 0.13 I switched to pnpm and wiped the npm lockfile, so I have no idea what it was!
